tests/test_bot_slack.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import node_monitor.load_config as c
+from node_monitor.bot_slack import SlackBot  
+from unittest.mock import Mock
+
+@pytest.fixture
+def mock_slack_client(mocker):
+    mock_client = Mock()
+    mocker.patch("slack_sdk.WebClient", return_value=mock_client)
+    return mock_client
+
+def test_send_message(mock_slack_client):
+    slack_bot = SlackBot(c.TOKEN_SLACK)
```

Comment:
The mocking implementation here is a lot different than what we have in `test_bot_email.py`. Whichever we choose to use, we need to make sure that we're consistent across files.
I chose not to use `pytest-mock` because I found pytest's syntax weird. I don't think we need a fixture either, we're not planning on running lots of tests, it adds boilerplate, and I find the syntax really unnatural.

If you can give clear justification on why we should use 
`pytest-mock` instead of just `from unittest.mock import patch` then we'll keep it. Otherwise, I think we should remove this. It just makes it more complex.

Reply:
You are right! I was going mainly off my own research but there is no added benefit. I switched it to be consistent with your implementation in commit [95b5dda](https://github.com/aviate-labs/node-monitor/pull/68/commits/95b5ddac5c05e5b1e18fbd34a543098e6f8378d6).